BuildResidentialHPXML/measure.rb

```diff
@@ -4153,6 +4205,10 @@ def self.set_hvac_distribution(hpxml, runner, args)
                                  conditioned_floor_area_served: args[:geometry_cfa],
                                  number_of_return_registers: number_of_return_registers)
 
+    if [HPXML::HVACTypeBoiler].include?(args[:heating_system_type]) && args[:heating_system_is_shared_system] && [HPXML::AirTypeFanCoil].include?(args[:heating_system_shared_distribution_type])
+      hpxml.hvac_distributions[-1].air_type = HPXML::AirTypeFanCoil
```

Comment:
This isn't right. If the heating system is a shared boiler w/ fan coil and the cooling system is a central AC, for example, this code would have them sharing a HPXML `HVACDistribution` of type fan coil, which doesn't make sense.

We really need to create a separate `HVACDistribution` element for a fan coil unit. As indicated by the name I suggested ("Boiler, Shared, w/ Ductless Fan Coil"), lets assume it is ductless for now. (It wouldn't make sense to use the same duct properties provided for the central AC in my example.) You'll have to set the supply and return duct leakages to zero and you can leave out any `Ducts`. I suggest creating this new distribution system as the last thing we do in the measure so that any existing code (like [here](https://github.com/NREL/OpenStudio-HPXML/blob/ed8001a70dc31f872581eae8165b2a2aea3bb4a8/BuildResidentialHPXML/measure.rb#L4312-L4319)) doesn't get confused by there possibly being two AirDistributions now.